client/views/room/MemberListRouter.js

```diff
@@ -6,6 +6,18 @@ import { useRoom } from './providers/RoomProvider';
 import { useTab, useTabBarClose, useTabContext } from './providers/ToolboxProvider';
 import RoomMembers from './contextualBar/RoomMembers';
 
+const getUid = (room, ownUserId) => {
+	if (room.uids.length === 1) {
+		return room.uids.length;
+	}
+
+	const uid = room.uids.filter((uid) => uid !== ownUserId).shift();
+
+	// Self DMs used to be created with the userId duplicated.
```

Comment:
Remove the comments on the code and use this nice information on the PR's description